### ISSUE TITLE
Feature Request: PNG images

### ISSUE THREAD

**Issue by raefa** [AppWerft/TiJSPDF#5]
TiJSPDF is already very hand. PNG images would make it even better! :)

**Comment by AppWerft** [comment]
Indeed. I think so. Perhaps I can integrate the module. Currently I'm using imageFactory to convert to JPEG outside of jspdf.

**Comment by raefa** [comment]
I am doing the same (converting images to JPEG) but it would be nice to have PNG's and transparencies so images could be overlaid. :)

**Comment by AppWerft** [comment]
I see the original png module uses 'Uint8Array' and other function that are not availabe in Titanium host. Like the main jspdf I have to rewrite and use Ti.Buffer udn Ti.Codec. It is a lot of work. Will see.
